src/Makefile.test.include

```diff
@@ -202,6 +202,7 @@ BITCOIN_TESTS += wallet/test/db_tests.cpp
 endif
 
 FUZZ_WALLET_SRC = \
+ wallet/test/fuzz/rpc.cpp \
```

Comment:
This change should be replaced with the following diff:
```diff
--- a/src/wallet/test/fuzz/CMakeLists.txt
+++ b/src/wallet/test/fuzz/CMakeLists.txt
@@ -10,6 +10,7 @@ target_sources(fuzz
     fees.cpp
     $<$<BOOL:${USE_SQLITE}>:${CMAKE_CURRENT_LIST_DIR}/notifications.cpp>
     parse_iso8601.cpp
+    rpc.cpp
     $<$<BOOL:${USE_SQLITE}>:${CMAKE_CURRENT_LIST_DIR}/scriptpubkeyman.cpp>
     wallet_bdb_parser.cpp
 )
```

Reply:
Thanks! 

Updated in [2c8cb2b](https://github.com/bitcoin/bitcoin/pull/30570/commits/2c8cb2b7cf9b6aa60562feb6aad2c6d071ee184b)